### DISCUSSION TITLE
Install options not appearing despite Runtime Compat being set

### DISCUSSION THREAD

**Discussion by nomadshiba** [jsr-io/jsr#769] [Q&A] [answered]
I noticed that the install options section isn't showing up on my package page: https://jsr.io/@nomadshiba/typed-contracts
I’ve made sure the "Runtime Compat" setting is set, but i can't figure out what might be causing the issue.
I’ve checked the docs but couldn’t find anything related.  

I have another package under a different scope where it works just fine.

**Comment by marvinhagemeister** [comment]
Converted this to an issue. The instructions not showing up is most likely a bug. Let's track it here https://github.com/jsr-io/jsr/issues/770

**Comment by nomadshiba** [comment, marked as answer]
Solved: https://github.com/jsr-io/jsr/issues/770#issuecomment-2438143146
